packages/boot/test/fast-usdc/fast-usdc.test.ts

```diff
@@ -92,13 +92,22 @@ test.serial(
     refreshAgoricNamesRemotes();
     t.truthy(agoricNamesRemotes.instance.fastUsdc);
     t.truthy(agoricNamesRemotes.brand.FastLP);
+    const lpAsset = agoricNamesRemotes.vbankAsset.FastLP;
+    t.like(lpAsset, {
+      issuerName: 'FastLP',
+      denom: 'ufastlp',
+      displayInfo: { assetKind: 'nat', decimalPlaces: 6 },
+    });
+    const lpId = lpAsset.brand.getBoardId();
+    t.is(agoricNamesRemotes.brand.FastLP.getBoardId(), lpId);
 
     const { EV } = t.context.runUtils;
     const agoricNames = await EV.vat('bootstrap').consumeItem('agoricNames');
     const board = await EV.vat('bootstrap').consumeItem('board');
     const getBoardAux = async name => {
       const brand = await EV(agoricNames).lookup('brand', name);
       const id = await EV(board).getId(brand);
+      t.is(id || null, lpId);
```

Comment:
I had a little trouble making sense of what it's testing. Casting all falsy `id` to `null` because `lpId` will either have a string or `null`? I can't tell why that's necessary.

please try to make it clearer

Reply:
yeah; that's goofy.

I fixed it so `lpId` is definitely a string instead of making the other side maybe null.